src/js/components/Ballot/CandidateItem.jsx

```diff
@@ -50,25 +49,25 @@ export default class Candidate extends Component {
                          alt="candidate-photo"/> :
                     <i className="icon-lg icon-main icon-icon-person-placeholder-6-1 icon-light utils-img-contain-glyph"/>
               }
-
+              {twitter_followers_count ?
```

Comment:
I wondered whether it made sense to move this into a component, but I just copied and pasted the snippet for now. I also noticed twitter counts were referred to a few different ways. (`twitter_followers_count`, `followers`, `twitterFollowers`) Assuming this could be cleaned up a bit.